osquery/killswitch/plugins/tests/killswitch_filesystem_tests.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in
+ the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2
+ (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <cstdint>
```

Comment:
nit: this is quite a few includes for just one test, are you sure you need all of these included?